### ISSUE TITLE
[Data@TAMU] Community page (All)

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#9]
**URL:** https://data-pre.library.tamu.edu/communities/27abe8e6-f875-43e8-a681-8dbdaa5128c7/search
**WCAG 2.1 AA compliant? :** No
**Violation:** 4.1.2 Name, Role, Value (A)
**Issues:**  Broken ARIA menu
**Notes:** There is a menubar with no items within it: `<div _ngcontent-dspace-angular-c2948902503="" role="menubar" class="dso-edit-menu d-flex">`
